content/blog/2023-02-09-ethereum-classic-course-8-ethereum-classics-monetary-policy/index.md

```diff
@@ -0,0 +1,153 @@
+
+It is described as “fair” because the coins are being issued and paid to people and entities who actually perform work for the network. Miners secure the network because their work imposes an enormous cost for any bad actors to tamper with the blockchain.
+
+Additionally, the fact that the currency is issued only after performing significant work makes it sound money as it is unforgeably costly and scarce.
+
+Block frequency and block rewards are the key components of the monetary policy because they determine the timing of the issuance, thus inflation rate and stock-to-flow ratio.
+
+For example, as the ETC blocks are produced every 15 seconds on average, and the current block rewards are 2.56 ETC, then, in this era, the rate of currency issuance is 5,382,144 per year. And, this rate of inflation will be reduced by 20% every 28 months.
+
+## 4. Uncle Blocks
+
+![There are, on average, 270,000 uncle blocks per era.](./5.png)
+
+As Ethereum Classic produces blocks every 15 seconds, instead of every ten minutes as Bitcoin, there are instances in which more than one valid block is produced by miners at any given time. However, only one block per round may be permitted, so to prevent any disincentive that miners may have for producing extra blocks in vain, there is a payment for these stale blocks, also called “uncle blocks”.
+
+Since inception, there has been a rate of production of uncle blocks of more or less 5.4%, which is approximately 270,000 uncle blocks per era.
+
+Uncle blocks were paid 4.375 ETC/ETH in the first era, but that payment was lowered to 0.125 ETC when the monetary policy was changed, and will be discounted, together with the normal block rewards, by 20% every 5,000,000 blocks.
+
+## 5. The Fifthening
+
+![Fifth-ening means a discount of 20% or 1/5.](./6.png)
+
+The event of the fifthening is celebrated by the ETC community every 28 months or 5,000,000 blocks and it happens on the date when the block of the next era is mined. 
+
+For example, we are in the 4th era and the 5th era will start on block 20,000,001, which will happen sometime in August of 2024.
+
+It is called the fifthening (fifth-ening) because the discount of 20% in each era is equivalent to one fifth of the block rewards and annual issuance.
+
+The importance of the fifthening is that the annual supply rate is lowered, making ETC sounder money as time passes by. This soundness is reflected in both a lower annualized inflation rate and a higher stock-to-flow ratio.
+
+## 6. Inflation Rate
+
+![ETC projected inflation rate by era compared with other assets.](./7.png)
+
+What is called the inflation rate is the rate of production or issuance of new currency per year. 
+
+When Ethereum Classic and Ethereum were one chain the annual rate of coin production was very high, at more than 14%, because the base was small and issuance was still high.
```

Comment:
Line 111: Punctuation changes; Remove unn\ecessary prepositiom
```suggestion
When Ethereum Classic and Ethereum were one chain, the annual rate of coin production was very high, more than 14%, because the base was small and issuance was still high.
```